commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -87,3 +92,58 @@ def run_filter(self, params, active_order_management_system, order_number):  # p
         except CommercetoolsError as err:  # pragma no cover
             log.exception(f"[{type(self).__name__}] Commercetools Error: {err}, {err.errors}")
             return PipelineCommand.CONTINUE.value
+
+
+class CreateReturnForCommercetoolsOrder(PipelineStep):
+    """
+    Creates refund/return for Commercetools order by Updating its
+    ReturnShipmentStatus & ReturnPaymentStatus
+    """
+
+    def run_filter(
+            self,
+            active_order_management_system,
+            order_number,
+            order_line_id
+    ):  # pylint: disable=arguments-differ
+        """
+        Execute a filter with the signature specified.
+        Args:
+            active_order_management_system: The Active Order System
+            order_number: Order number (for now this is an order.id, but this should change in the future)
+            TODO: SONIC-277 (in-progress)
+            order_line_id: ID of order line item
+        Returns:
+            returned_order: Updated Commercetools order
+            returned_order_line_item_id: Updated Commercetools order's return item ID
```

Comment:
Remember to also update this to 'returned_line_item_return_id' as per your return statement